Tests/SKCoreTests/FileBuildSettingsTests.swift

```diff
@@ -0,0 +1,29 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@testable import SKCore
```

Comment:
We don't allow `@testable` imports, because it requires building in a way that's not compatible with how we want to distribute our binary.  Feel free to make the API public to test it, but please (1) add an underscore, and (2) add "public for testing" at the front of its documentation comment.